src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -231,28 +324,66 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
       applyThemesOnElement(this, this.hass.themes, this._config.theme);
     }
 
+    if (!this._progressInterval && this._showProgressBar) {
```

Comment:
Showing a progress bar AND is playing.

Reply:
And not narrow.

Reply:
Not Narrow is in the show progress bar getter